luma/lcd/const.py

```diff
@@ -6,3 +6,8 @@
 class pcd8544(object):
     DISPLAYON = 0x0C
     DISPLAYOFF = 0x08
+
+
+class st7735(object):
```

Comment:
ok cool, it's more that I meant to say we could use this directive for any new classes or methods:

```
.. versionadded:: 3.5
```